packages/core/src/card/card.stories.ts

```diff
@@ -0,0 +1,150 @@
+    Suspendisse non interdum sem, nec egestas nulla.
+    Duis vitae leo justo. Fusce ante enim, tincidunt nec faucibus sed, fringilla et ipsum.
+    Proin quis diam elit. Cras nec finibus libero. Sed bibendum lectus eget magna ultricies eleifend.
+    Duis ut molestie urna. Proin cursus dolor ac ex rutrum, eu condimentum elit tincidunt.
+    In a dolor faucibus, vehicula ipsum eu, viverra lorem. Aenean congue fermentum commodo.
+    Sed ex purus, consectetur mollis urna eget, vestibulum mollis dolor.
+    Donec sodales, felis sit amet gravida ornare, magna dolor ullamcorper turpis, vel facilisis odio ipsum semper sem.
+  `;
+
+export default {
+  title: 'Stories/Card',
+  component: 'cds-card',
+  argTypes: getElementStorybookArgTypes('cds-card', customElements),
+  parameters: {
+    options: { showPanel: true },
+  },
+};
+
+export function Basic() {
+  return html`<cds-card>
+    <p cds-text="p1">${placeholder}</p>
+  </cds-card>`;
+}
+
+export function WithLayout() {
+  return html` <cds-card>
+    <div cds-layout="vertical">
+      <div cds-layout="p-t:lg p-l:lg">
+        <p cds-text="h4">Card Title</p>
+      </div>
+
+      <div cds-layout="p:lg">
+        <p cds-text="p1">${basicContent}</p>
+      </div>
+
+      <cds-divider></cds-divider>
+
+      <div cds-layout="p:lg"></div>
+    </div>
+  </cds-card>`;
+}
+
+export function WithLayoutAndTwoDividers() {
+  return html` <cds-card>
+    <div cds-layout="vertical">
+      <div cds-layout="p:lg">
+        <p cds-text="h4">Card Title</p>
+      </div>
+
+      <cds-divider></cds-divider>
+
+      <div cds-layout="p:lg">
+        <p cds-text="p1">${basicContent}</p>
+      </div>
+
+      <cds-divider></cds-divider>
+
+      <div cds-layout="p:lg"></div>
+    </div>
+  </cds-card>`;
+}
+
+export function WithLayoutTwoDividersAndButtons() {
+  return html` <cds-card>
+    <div cds-layout="vertical">
+      <div cds-layout="p:lg">
+        <p cds-text="h4">Card Title</p>
+      </div>
+
+      <cds-divider></cds-divider>
+
+      <div cds-layout="p:lg">
+        <p cds-text="p1">${basicContent}</p>
+      </div>
+
+      <cds-divider></cds-divider>
+
+      <div cds-layout="p-x:lg p-y:md">
+        <cds-button action="flat">View</cds-button>
+        <cds-button> View </cds-button>
+        <cds-button status="success"> View Success </cds-button>
+        <cds-button status="danger"> View Danger </cds-button>
+      </div>
+    </div>
+  </cds-card>`;
+}
+
+export function WithLayoutAndOverflow() {
+  return html` <style>
+      cds-card#cds-card-with-fixed-height .body {
+        height: 5rem;
+        overflow: auto;
+      }
+    </style>
+    <cds-card id="cds-card-with-fixed-height">
+      <div cds-layout="vertical">
+        <div cds-layout="p:lg">
+          <p cds-text="h4">Card Title</p>
+        </div>
+
+        <cds-divider></cds-divider>
+
+        <div cds-layout="p:lg" cds-text="p1" height="100px">
```

Comment:
It looks like this should have a `class="body"` on it to make the example work with overflow scrolling.